wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -252,10 +318,29 @@ class CreateSecureGuestLinkViewController: UIViewController, CreatePasswordSecur
             createSecureGuestLinkPasswordValidatorHelper.resetPasswordDefaultState(for: [textField], for: labels)
         } else {
             createSecureGuestLinkPasswordValidatorHelper.displayPasswordErrorState(for: [textField], for: labels)
+            announcePasswordValidationErrorForVoiceOver(for: textField)
         }
 
         return isValid
     }
+
+    // MARK: - Accessibility
+
+    func announcePasswordValidationErrorForVoiceOver(for textField: ValidatedTextField) {
+        if textField == securedGuestLinkPasswordTextfield {
+            UIAccessibility.post(
+                notification: .announcement,
+                argument: SecureGuestLinkAccessibilityLocale.SecuredGuestLinkPasswordTextfield.announcement
+            )
+        } else {
+            UIAccessibility.post(
+                notification: .announcement,
+                argument: SecureGuestLinkAccessibilityLocale.SecuredGuestLinkPasswordValidatedTextField.announcement
+            )
+        }
```

Comment:
readability improvement + if there's a way to make this `SecureGuestLinkAccessibilityLocale.SecuredGuestLinkPasswordTextfield.announcement` shorter :)
 
```suggestion
    func announcePasswordValidationErrorForVoiceOver(for textField: ValidatedTextField) {
       let argument =  textField == securedGuestLinkPasswordTextfield ? SecureGuestLinkAccessibilityLocale.SecuredGuestLinkPasswordTextfield.announcement : SecureGuestLinkAccessibilityLocale.SecuredGuestLinkPasswordValidatedTextField.announcement
        
            UIAccessibility.post(
                notification: .announcement,
                argument: argument
            )
        }
```